diff --git a/.sanctum_patch_applied b/.sanctum_patch_applied
new file mode 100644
index 0000000..3d1f86b
--- /dev/null
+++ b/.sanctum_patch_applied
@@ -0,0 +1 @@
+Patched with patch file dating back to revision 50660ebd41e189051592bcb2d778a645f9e67b7c of git@github.com:ilebedev/sanctum.git
diff --git a/Makefile.in b/Makefile.in
index f4d771c..5cb66ef 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -1,3 +1,5 @@
+GDB_TARGET_FLAGS_EXTRA := --enable-gui
+
 srcdir := @abs_top_srcdir@
 builddir := @abs_top_builddir@
 INSTALL_DIR := @prefix@
